gcp/appengine/main.py

```diff
@@ -14,14 +14,15 @@
 """App Engine entrypoint."""
 
 import logging
-import os
 
 from flask import Flask
 import google.cloud.logging
 from google.cloud import ndb
 
+from cache import cache
```

Comment:
I think this is importing a variable from a module which I beleive is against the styleguide:
https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Great point, done!